test/replicaset-luatest/vconnect_test.lua

```diff
@@ -176,8 +176,12 @@ test_group.test_async_no_yield = function(g)
     local csw1 = fiber.self():csw()
     local ret, err = rs:callrw('get_uuid', {}, opts)
     local csw2 = fiber.self():csw()
-    -- Waiting for #456 to be fixed.
-    t.assert_equals(csw2, csw1 + 1)
+    if ivutil.version_is_at_least(2, 11, 0, nil, 0, 0) then
```

Comment:
Please, include `ivutil`

Reply:
AFAIC, `ivutil` is globally available in tests. But this test already requires `vutil`, so I just changed `ivutil` to `vutil`.

Reply:
Thank you